Sources/Common/Service/HttpRequestRunner.swift

```diff
@@ -56,10 +56,11 @@ public class UrlRequestHttpRequestRunner: HttpRequestRunner {
         let directoryURL = fileType.directoryToSaveFiles(fileManager: FileManager.default)
 
         session.downloadTask(with: url) { tempLocation, response, _ in
-            guard let tempLocation = tempLocation, let uniqueFileName = response?.suggestedFilename else {
+            guard let tempLocation = tempLocation, let suggestedFileName = response?.suggestedFilename else {
                 return onComplete(nil)
             }
 
+            let uniqueFileName = UUID().uuidString + "_" + suggestedFileName
```

Comment:
I think we should have a comment in the code explaining why having a unique file name is important. Or, why this UUID is needed beyond "suggest file name"

Reply:
i would have hoped naming would have done the trick, as in `suggestedFileName` instead of `uniqueFileName` which was before, because suggested file name even from the name doesn't say its going to be unique. 

and then adding `UUID().uuidString` and referring it as `uniqueFileName` to mention we would have a unique path. 

So from a presepective of someone who is looking for the first time, they would understand whats happening just by looking into names.
 But i can add comment on it, if it helps.